hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/alias/CredentialShell.java

```diff
@@ -66,6 +67,7 @@
    * <pre>
    * % hadoop credential create alias [-provider providerPath]
    * % hadoop credential list [-provider providerPath]
+   * % hadoop credential check alias [-provider providerPath]
```

Comment:
should the full usage option set be listed here?

Reply:
> should the full usage option set be listed here?

Just to keep the format uniform and `[-value]` is for test, may not need here.